containers/docs/setup-and-install.md

```diff
@@ -0,0 +1,132 @@
+## Setup
+
+This setup is done in two phases: build and deployment. The build phase is handled by the `ansible-container` project using Ansible roles to build out each container in preparation to push to Openshift. The deployment phase can either be done locally or to a local or remote Openshift cluster. This README will walk through how to build and run on Openshift:
+
+  1. [Install ansible](#install-ansible)
+  2. [Install and Configure Docker](#setup-docker)
+  3. [Install ansible-container](#install-ansible-container)
+  4. [Build Containers](#build-containers)
+  5. [Run Containers Locally](#run-containers-locally)
+  6. [Setup and Install Local OpenShift](#setup-and-install-openshift)
+  7. [Deploy Containers to OpenShift](#deploy-containers-to-openshift)
+
+This installation guide assumes that you are working from the `forklift/containers` directory for all actions.
+
+### Install Ansible
+
+Parts of the setup have been automated using Ansible to ensure consistency and speed up getting to the deployment aspect. Please ensure you have the latest version of Ansible installed:
+
+Using yum:
+
+    sudo yum install ansible
+
+Using pip:
+
+    sudo pip install ansible
+
+### Setup Docker
+
+In order to build the containers, run them locally or use OpenShift, Docker needs to be installed and configured properly. First, ensure that docker is installed:
+
+    sudo yum install docker
+
+Second, we need to allow for insecure registries to run. This can be enabled by editing `/etc/sysconfig/docker` and adding:
+
+    INSECURE_REGISTRY='--insecure-registry 172.30.0.0/16'
+
+Now restart the docker service:
+
+    sudo systemctl restart docker
+
+### Install ansible-container
+
+This project is currently using the bleeding edge version of `ansible-container` and will be installed from source. To simplify install, an Ansible playbook has been provided to do the installation:
+
+    ansible-playbook install-ansible-container.yml
+
+### Build Containers
+
+Now that `ansible-container` is installed, the containers can be built. This build takes a while and can cause the docker client to timeout. Thus, before building run:
+
+    export DOCKER_CLIENT_TIMEOUT=600
+
+Now build the containers (and grab a coffee):
+
+    ansible-container build
+
+### Run Containers Locally
+
+Once the containers have been built locally into Docker, `ansible-container` can be used to run the entire stack locally:
+
+    ansible-container run
```

Comment:
I should just rip this out for now. The `run` won't work currently due to port overlap and secrets support missing.